tests/cases/interfaces/name-interface

```diff
@@ -0,0 +1,18 @@
+---DESCRIPTION---
+Forbid Interface suffix
```

Comment:
I think `Interface` and `Abstract` have been in use before and hence are worthy of forbidden. `Exception` is easy to forbid, but people can just name everything `*Problem` or `*Error` so doesn't necessarily have a positive effect.

Reply:
We can add more rules as we need. Naming will always require a human review.